### PR TITLE
Updated balancer EA READMEs with descriptions for MAX_PAYLOAD_SIZE_LIMIT

### PR DESCRIPTION
**Pull request by karen-stepanyan**
[DF-20246](https://smartcontract-it.atlassian.net/browse/DF-20246)

Added `known issues` section in README files for balancer EAs that explains the connection between `MAX_PAYLOAD_SIZE_LIMIT` env var the EA supports and Nginx config. 

Note for reviewer. 
Ideally this should apply to all EAs, although is currently relevant to balancer EAs as they accept an array of any size as an input parameter. I'm open to include this description in the [ea-framework](https://github.com/smartcontractkit/ea-framework-js/blob/main/src/config/index.ts#L165) itself, but since the ea-framework configs are not linked to EA READMEs, I'm also ok having those explicitly in balancer EAs READMEs for now

[DF-20246]: https://smartcontract-it.atlassian.net/browse/DF-20246?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by mxiao-cll** [comment]
Can we just put content in one file and point to that file?

**Comment by karen-stepanyan** [comment]
@mxiao-cll As I mentioned in the PR description, ideally we should have the content in a ea-framework repo and just point to it from EA READMEs, there is a separate ticket for that. Until that is done we can have the content here for each README

**Review by mxiao-cll — APPROVED** [review]

File: packages/sources/amberdata/docs/known-issues.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+## Known Issues
```
Comment (mmcallister-cll) [inline packages/sources/amberdata/docs/known-issues.md:1]: README update missing here

**Comment by mmcallister-cll** [comment]
are all of these EAs related to growing address lists and/or PoR feeds?

File: packages/sources/amberdata/docs/known-issues.md (hunk 1 above)
Comment (austinborn) [reply packages/sources/amberdata/docs/known-issues.md:1]: What do you mean? That the `amberdata` README should point to this?

**Review by austinborn — APPROVED** [review]

**Comment by karen-stepanyan** [comment]
@mmcallister-cll Yes. It seems that i've missed couple but don't think they are used that much.

**Review by mmcallister-cll — APPROVED** [review]

**Comment by mmcallister-cll** [comment]
Yeah I think we're good with this set, can always move to the others as well

**Pull request merged**